lib/cdk-infra-stack.ts

```diff
@@ -715,7 +968,7 @@ export default class FhirWorksStack extends Stack {
             ?.addResource('.well-known')
             .addResource('smart-configuration')
             .addMethod('GET', new LambdaIntegration(fhirServerLambda), {
-                authorizationType: AuthorizationType.NONE,
+                authorizationType: AuthorizationType.CUSTOM,
```

Comment:
Why are these endpoint authorizationTypes being changed to CUSTOM?

Reply:
cfn-nag warns that they shouldnt be NONE. CUSTOM was the most fitting, as we do have a separate authorization

Reply:
created!